### PR TITLE
feat(cli): better new line mode

### LINKED ISSUE #145: bug: display error
```
root@localhost:8000/> select 'a\nb' as a, hex(a);

SELECT
  'a\nb' AS a,
  hex(a)

┌─────────────────┐
│    a   │ hex(a) │
│ String │ String │
├────────┼────────┤
│ a      │ 610a62 │
│ b      │        │
└─────────────────┘
1 row in 0.007 sec. Processed 1 rows, 1B (149.15 rows/s, 149B/s)


```

mysql client:
```
mysql> select 'a\nb' as a, hex(a);
+------+--------+
| a    | hex(a) |
+------+--------+
| a
b  | 610a62 |
+------+--------+
1 row in set (0.00 sec)
Read 1 rows, 1.00 B in 0.001 sec., 999.27 rows/sec., 999.27 B/sec.
```

### PR DESCRIPTION
**Pull request by sundy-li**
fixes #145

### CODE REVIEW

**Comment by everpcpc** [comment]
you could use `scripts/bump.sh 0.4.4`

File: cli/src/display.rs [hunk 1]
```diff
@@ -424,7 +434,11 @@ fn create_table(
         let values = row.values();
         let mut v = vec![];
         for value in values {
-            v.push(value.to_string());
+            if replace_newline {
+                v.push(value.to_string().replace('\n', "\\n"));
```
Comment (everpcpc) [inline cli/src/display.rs:438]: should `\r` also be replaced?

Comment (everpcpc) [reply cli/src/display.rs:438]: and what about control characters like `\b` or `\v`

Comment (sundy-li) [reply cli/src/display.rs:438]: I think just ‘\n' is ok, this is to control the display.

**Review by TCeason — APPROVED** [review]

**Pull request merged**